### PR TITLE
Masks in rollouts storage should start with zeros, not ones.

### PR DESCRIPTION
**Pull request by Lucaweihs**
The masks are defined to be 0 when done, otherwise 1. Namely, in the engine we have:
```python
        masks = torch.tensor(
            [0.0 if done else 1.0 for done in dones],
            dtype=torch.float32,
            device=self.device,  # type:ignore
        ).view(
            -1, 1
        )  # [sampler, 1]
```
This means that the initial input to the rollout storage should have the masks all be zero (so that hidden states to can be reset to whatever their default values are).

### CODE REVIEW

**Comment by jordis-ai2** [comment]
LGTM. Feel free to merge 👍
